includes/staticrequesthandler.php

```diff
@@ -18,19 +17,26 @@
     $url = $https ? 'https://' : 'http://';
     $url .= $host . $uri;
 
-
     if (is_callable($urlMapping)) {
         $path = $urlMapping($url);
     } else {
         $path = URLtoPath($url);
     }
 
-    if (!$path) {
+    if (! $path) {
         return false;
     }
 
     $cachePath = $cacheDir . DIRECTORY_SEPARATOR . $path;
-
+    $hasHTMLFile = false;
+    $hasGZIPFile = false;
+    if (file_exists($cachePath . '.html')) {
+        $hasHTMLFile = true;
+        $cachePath .= '.html';
+    } elseif (file_exists($cachePath . '.html.gz')) {
```

Comment:
Should it not check for the gzip first and serve that as a priority? Could both exist at the same time?